content/using/planet.md

```diff
@@ -3,7 +3,7 @@ title = "Setting up Your Planet"
 weight = 1
 description = "Getting started with a permanent Urbit identity."
 template = "page_indiced.html"
-aliases = ["/using/install.md#boot-your-planet", "/using/install.md#the-dojo"]
+aliases = ["@/using/install.md#boot-your-planet", "@/using/install.md#the-dojo"]
```

Comment:
Not how this works. The reason internal link format uses .md is that it scours the actual files and directories.

Reply:
Oh, oops. I got a false positive that it worked on my machine. But I had changed the linking file